src/transformers/pipelines/audio_utils.py

```diff
@@ -50,6 +50,7 @@ def ffmpeg_microphone(
     sampling_rate: int,
     chunk_length_s: float,
     format_for_conversion: str = "f32le",
+    ffmpeg_input_device: Optional[str] = None,
```

Comment:
It would be nice if we could also add a documentation to the `ffmpeg_microphone` method.

Reply:
+1